README.md

```diff
@@ -0,0 +1,19 @@
+# Source code for `https://pest.rs`
+
+This repo contains the source code for https://pest.rs
+
+You can build it with [wasm-pack](https://github.com/rustwasm/wasm-pack):
+
+```sh
+# Create an empty directory to hold the output (and remove any old files)
+mkdir -p www
+rm -rf www/*
+
+# Build and copy in the JS and WASM from Pest
+wasm-pack build --taget web
+cp pkg/*.js www/
+cp pkg/*.wasm www/
+
+# Copy in authored files
+cp -R static/* www/
+```
```

Comment:
Maybe add that for testing, you can serve `www` with any ad-hoc static file server like `python -m http.server`?